src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -797,12 +816,12 @@ namespace InternalHeatGains {
                                     People(Loop).AngleFactorListName = AlphaName(8);
 
                                 } else if (mrtType == "") { // Blank input field--just ignore this
-                                    if (MustInpSch && Item1 == 1 && (People(Loop).Fanger || People(Loop).Pierce || People(Loop).KSU))
+                                    if (MustInpSch && Item1 == 1 && ThermalComfortRuns)
```

Comment:
I don't see the point of `MustInpSch`. I know it's in the old code, but it's true for *every* thermal comfort option, so it seems redundant. Maybe you can see how it's useful.

Reply:
`MustInpSch` is not used and deleted.